### PR TITLE
Backtesting-till-now works with most recent candle instead of until last midnight

### PR DESCRIPTION
**Pull request by rhulshof**
Plotly did not open the chart by default anymore. This commit is only to stick with the previous behavior.

### CODE REVIEW

**Review by sheridavandenbent — DISMISSED** [review]
Good catch, thank you!

**Comment by rhulshof** [comment]
Another commit was added. Since this pull request appeared to be still open; the commit was appended. Nevertheless, both are worth pulling :)

**Review by geordle — CHANGES_REQUESTED** [review]

**Comment by rhulshof** [comment]
1. It still states changes are requested, whilst I already committed the requested changes. Am I missing something or is this PR ready to be merged now?
2. Interesting why the pytest-test keeps failing? Something I am doing wrong here?

File: modules/output/plots_per_coin.py [hunk 1]
```diff
@@ -63,4 +63,4 @@ def plot_coin(config, stats, pair: str, pair_data):
         os.remove("data/backtesting-data/binance/plot%s.html" % pair.replace("/", ""))
     except OSError:
         pass
-    fig.write_html("data/backtesting-data/plots/plot%s.html" % pair.replace("/", ""), config={'scrollZoom': True})
+    fig.write_html("data/backtesting-data/plots/plot%s.html" % pair.replace("/", ""), config={'scrollZoom': True}, auto_open=True)
```
[the remark this answers is not in the document]
Comment (geordle) [reply modules/output/plots_per_coin.py:66]: Would you mind still removing this bit right here?

```suggestion
    fig.write_html("data/backtesting-data/plots/plot%s.html" % pair.replace("/", ""), config={'scrollZoom': True})
```

**Review by geordle — APPROVED** [review]

**Pull request merged**